middlewares/authorize.middleware.js

```diff
@@ -1,7 +1,7 @@
 const jwt = require("jsonwebtoken");
 
 module.exports = function (req, res, next) {
-  let token = req.header("authorization");
```

Comment:
@ashabhussan bhai, `let token = req.header("authorization");` is that properly working..?